Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs

```diff
@@ -26,6 +27,17 @@ internal class IosBrowserAuthBroker : NSObject, IBrowserAuthBroker, ISignInNavig
         private SignInViewController _signInViewController;
         private bool _isTaskComplete;
         private string _endUrlString;
+        private readonly HashSet<string> _browserLinks = new HashSet<string>
```

Comment:
Most of these urls seem a little too specific.  The clcid and region, mobile=false and that vv one all seem like they shouldn't be part of the string.  I expect we'll need to use something like .StartsWith comparisons to be robust here.